### PR TITLE
Update architecture.adoc

### PR DESCRIPTION
**Pull request by anilkumar29**

### CODE REVIEW

File: docs/modules/ROOT/pages/architecture.adoc [hunk 1]
```diff
@@ -4,17 +4,17 @@ ifdef::env-github[]
...
+Couchbase monitoring and observability stack [CMOS] is built primarily with Grafana-based components mainly for the following reasons:
 
 * Already in use with Prometheus exporter and similar on Couchbase Server
 * Industry standard and OSS with various integration points for other pipelines
 * Existing options to scale from single node up to cloud scale easily
 
-We expose Prometheus endpoints already for node-level information on a Couchbase Server instance.
-Recently we have exposed logs using Fluent Bit, primarily for kubernetes based solutions but this can be deployed on-premise as well.
+We expose Prometheus endpoints already for node-level information on a Couchbase Server instance starting from Couchbase Server 7.0 release.
+Recently we have exposed logs using Fluent Bit, primarily for Kubernetes based solutions but this can be deployed on-premise deployment as well.
```
Comment (maria-robobug) [inline docs/modules/ROOT/pages/architecture.adoc:14]: Find this a bit confusing with the addition of `deployment` at the end, though I may not have enough understanding of the CMOS setup. Previous wording seemed fine, aside from Kubernetes not being capitalized ofcourse.

**Review by maria-robobug — APPROVED** [review]
👍 just minor comment.

File: docs/modules/ROOT/pages/architecture.adoc [hunk 2]
```diff
@@ -4,17 +4,17 @@ ifdef::env-github[]
...
 
-Couchbase have a proprietary project that provides cluster-level information via a Prometheus endpoint: https://github.com/couchbaselabs/cbmultimanager
-This integrates Couchbase knowledge and best practices into a reusable component. We can then reuse this component in the monitoring stack here as it provides a Prometheus endpoint.
+One of the core capabilities of CMOS is the Couchbase Cluster Monitor that provides cluster-level information via a Prometheus endpoint: https://github.com/couchbaselabs/cbmultimanager.
+Couchbase Cluster Monitor is a closed-source enterprise-only solution that integrates Couchbase knowledge and best practices into a reusable component.
```
Comment (markspolakovs) [inline docs/modules/ROOT/pages/architecture.adoc:17]: Not sure if we should link to a private repository in our public docs?

**Review by markspolakovs — APPROVED** [review]
LGTM with a minor nitpick :+1:

File: docs/modules/ROOT/pages/architecture.adoc (hunk 1 above)
Comment (anilkumar29) [reply docs/modules/ROOT/pages/architecture.adoc:14]: I agree @maria-robobug I'm updating the statements.

File: docs/modules/ROOT/pages/architecture.adoc (hunk 2 above)
Comment (anilkumar29) [reply docs/modules/ROOT/pages/architecture.adoc:17]: I missed it. You're right we shouldn't.

**Pull request merged**